internal/cli/cmd/connectivity.go

```diff
@@ -119,6 +119,7 @@ func newCmdConnectivityTest() *cobra.Command {
 	cmd.Flags().BoolVarP(&params.Verbose, "verbose", "v", false, "Show informational messages and don't buffer any lines")
 	cmd.Flags().BoolVarP(&params.Debug, "debug", "d", false, "Show debug messages")
 	cmd.Flags().BoolVarP(&params.PauseOnFail, "pause-on-fail", "p", false, "Pause execution on test failure")
+	cmd.Flags().BoolVar(&params.SkipIPCacheCheck, "skip-ip-cache-check", true, "Skip IPCache check")
```

Comment:
Should we mark this flag as hidden so we can easily remove it again? Given that we intend to fix the ipcache flake #361 eventually,

Reply:
Hmmm this makes sense, but at the same time I think that if users face the issue and the flag is hidden they won't know they can use it to proceed.

Reply:
Given that we default the flag to `true` (i.e. skip the ipcache check by default), I think users should not face the ipcache issue anymore. They would need to figure out to run with `--skip-ip-cache-check=false` at which point they already figured out the flag :wink:

Reply:
Oh right, makes total sense 💯 — I hadn't looked at the default.

Reply:
Looks like this hasn't been addressed before merging. I've opened another PR: #508

Reply:
thanks @tklauser , i just saw the green merge button and pushed it 😅